app/src/main/java/com/architectcoders/aacboard/ui/fragments/adapter/PictogramsSearchAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package com.architectcoders.aacboard.ui.fragments.adapter
+
+import android.annotation.SuppressLint
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.architectcoders.aacboard.data.PictogramUI
+import com.architectcoders.aacboard.databinding.ItemSearchPictogramBinding
+import com.architectcoders.aacboard.ui.utils.loadUrl
+
+class PictogramsSearchAdapter(
+    private val onPictogramClicked: (PictogramUI?) -> Unit,
+) : RecyclerView.Adapter<PictogramsSearchAdapter.ViewHolder>() {
+
+    private var cells = emptyList<PictogramUI>()
+
+    @SuppressLint("NotifyDataSetChanged")
+    fun updateItems(newItems: List<PictogramUI>) {
+        cells = newItems
+        notifyDataSetChanged()
+    }
```

Comment:
Here you could use the `basicDiffUtil` instead of the notifyDataSetChanged. There is an example of use in the `DashboardCellsAdapter`

Reply:
Ok. I will update it (and thanks for the tip)